deployer/src/deploy-struct.ts

```diff
@@ -131,6 +131,7 @@ export interface DeployStructure {
     cleanNamespace?: boolean // Clears entire namespace prior to deploying
     bucket?: BucketSpec // Information guiding deployment of web resources into an s3 (or s3-like) object store bucket
     actionWrapPackage?: string // The name of a package into which web resources will be action-wrapped.
+    actionWrapPackageWithRouter?: boolean // The name of a package into which web resources will be action-wrapped.
```

Comment:
I don’t think we need a control flag that’s user visible. We should just do it this way by default.

Reply:
ok, will make the changes.

Reply:
There is a question of routing we might need to settle. Will follow up briefly in slack first.